source/_integrations/kaleidescape.markdown

```diff
@@ -0,0 +1,396 @@
+---
+title: Kaleidescape
+description: Instructions on how to integrate Kaleidescape into Home Assistant.
+ha_category:
+  - Media Player
+ha_release: '2021.10.7'
```

Comment:
The integration isn't released, yet it's already availalble for the last 4 months? 👀

Reply:
Good catch. I have had it on HACS for awhile letting some devices owners test it. Will update.